.rubocop.yml

```diff
@@ -17,68 +17,68 @@ AllCops:
     - rbx
   # Include common Ruby source files.
   Include:
-    - '**/*.rb'
```

Comment:
Please replace all double quotes for singles, this way it won't show every line as changed and will be able to check the real changes (maybe there aren't)

Reply:
I have not changed this type of configuration. I'm gonna check it!